site/content/ytt/docs/latest/how-to-export-schema.md

```diff
@@ -114,6 +121,12 @@ components:
 
 Of the properties declared in the [OpenAPI Schema specification](https://swagger.io/specification/#schema-object), the following are generated.
 
+### `title`
```

Comment:
```suggestion
### `title`

Available in v0.39.0+

```

Reply:
@vmunishwar following our discussion today about version caveats, can we add this? I got the inspiration from where where have done a version caveat [elsewhere](https://github.com/vmware-tanzu/carvel/blob/develop/site/content/ytt/docs/latest/lang-ref-ytt-library.md?plain=1#L237-L239)